lib/brains/forms.py

```diff
@@ -167,3 +169,44 @@ def clean(self):
             raise forms.ValidationError(
                 'Did not pass verification', code='invalid')
         return self.cleaned_data
+
+
+class PayMethodUpdateForm(forms.Form):
+    active = forms.BooleanField(required=False)
+
+    def __init__(self, data, obj):
+        self.object = obj
+        return super(PayMethodUpdateForm, self).__init__(data)
+
+    def clean_active(self):
+        active = self.cleaned_data['active']
+
+        # An attempt to enable active on the payment method.
+        if active and not self.object.active:
+            raise forms.ValidationError(
+                'Cannot set a payment method to active', code='invalid')
```

Comment:
"Cannot set an inactive payment method to active" or similar might be tiny bit clearer.